docs/quickstart.rst

```diff
@@ -26,30 +26,38 @@ To check if your endpoint/client have network access and can connect to the func
 
   >>> curl https://api2.funcx.org/v2/version
 
-This should return a version string, for example: ``"0.3.0"``
+This should return a version string, for example: ``"1.0.2"``
 
 .. note:: The funcx client is supported on MacOS, Linux, and Windows. The funcx-endpoint
    is only supported on Linux.
 
-Installation using Pip
-^^^^^^^^^^^^^^^^^^^^^^
+Installing funcX in a Virtual Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 While ``pip`` and ``pip3`` can be used to install funcX we suggest the following approach
-for reliable installation when many Python environments are available.
+for reliable installation to avoid python package dependency conflicts.
 
-1. Install the funcX client::
+1. Install the funcX client in its own `venv <https://docs.python.org/3/tutorial/venv.html>`_ environment::
 
-     $ python3 -m pip install funcx
+    $ python3 -m venv path/to/funcx_venv
+    $ source path/to/funcx_venv/bin/activate
+    (funcx_venv) $ python3 -m pip install funcx
 
-To update a previously installed funcX to a newer version, use: ``python3 -m pip install -U funcx``
+  To update a previously installed funcX to a newer version in the virtual environment, use::
 
-2. Optionally install the funcX endpoint::
+    (funcx_venv) $ python3 -m pip install -U funcx``
 
-     $ python3 -m pip install funcx_endpoint
+2. (Optional) The funcX endpoint can be installed using `Pipx <https://pypa.github.io/pipx/installation/>`_ or using pip in the venv::
```

Comment:
> or using pip in the venv

Hmm, since we suggest `pipx` above and elsewhere, this may be a dangerous _or_ clause.  Suggest sticking with the `pipx` installation directive in the documentation, else we'll be fielding reports from folks who take that out of context and miss the "_... in the venv_" part.

Reply:
"pip in the venv" plus the (funcx_venv) prefix to the command are redundant directives, so I don't think it's easily mistaken with regular pip.  Plus, Jupyter would need to be in the venv also with the sdk, so I think it's ok.

If you really feel strongly, we will have to reword both sections.  As pipx only works for the endpoint.